Gemfile

```diff
@@ -4,39 +4,13 @@ source "https://rubygems.org"
 gemspec
 
 gem "pg", "~> 1.0", platform: :ruby
-gem "activerecord-jdbcpostgresql-adapter", platform: :jruby
-gem "ffi-geos", platform: :jruby
 gem "byebug" if ENV["BYEBUG"]
-
-def activerecord_version
-  return ENV["AR_VERSION"] if ENV["AR_VERSION"]
-
-  require "uri"
-  require "yaml"
-  require "net/http"
-
-  # read gemspec to get activerecord version spec
-  # pull all activerecord versions from rubygems
-  # find the newest version that matches the version from our gemspec
-  gs = Bundler.load_gemspec("activerecord-postgis-adapter.gemspec")
-  ar_dep = gs.dependencies.find { |d| d.name == "activerecord" }
-
-  uri = URI("https://rubygems.org/api/v1/versions/activerecord.yaml")
-  res = Net::HTTP.get_response(uri)
-  versions = YAML.safe_load(res.body)
-  ver = versions.find { |v| ar_dep.match?("activerecord", v["number"]) }
-
-  raise Bundler::GemNotFound, "No matching Activerecord version found for #{ar_dep.requirement}" unless ver
-
-  ver["number"]
-end
```

Comment:
Any reason we're removing this? This is required in order to test against the full activerecord test suite. I know it's not part of the full CI yet, but IMO it's helpful to be able to run locally and potentially flag any issues.

Reply:
We still pulling, just using a static tag instead.

Reply:
This was needed when the adapter was supporting multiple versions (prior 5.1).
The CI never had a AR_VERSION set.

Reply:
I'm not talking about AR_VERSION but the dynamic part below (fetching the latest ar patch!)